recipes/nimfa/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "nimfa" %}
+{% set version = "1.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 39cff2b86856d03ca8a3d9c38598034ecf1a768c325fd3a728bb9eadb8c6b919
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7|>=3.4
+  run:
+    - numpy >=1.7.0
+    - python >=2.7|>=3.4
```

Comment:
```suggestion
    - python =2.7,>=3.4
```

Reply:
Shouldn't this be `python =2.7|>=3.4` like above?

Reply:
Yes 😅